src/opnsense/www/js/opnsense_bootgrid.js

```diff
@@ -843,6 +855,21 @@ class UIBootgrid {
             $last_btn.show();
         }
 
+        // Tree does not have a groupStartOpen style hook, so we must restore expansion after data is loaded
+        if (this.table?.options?.dataTree && this.rememberedTreeIds?.size) {
```

Comment:
some of the optional chaining operators can be scrapped here, overuse of this may hide mistakes in code we cannot easily spot otherwise (i.e. `rememberedTreeIds.size` should always exist, if it doesn't, it's not a `Set` - which wouldn't be valid here. )